cache-config/testing/ort-tests/tcdata/statuses.go

```diff
@@ -17,30 +17,14 @@ package tcdata
 
```

Comment:
Are the two functions in this file ever called anymore with all of the removals you did? I wouldn't think so, but I could be wrong.

Reply:
You're right, I've removed the file now.